packages/ts-migrate-plugins/tests/src/explicit-any.test.ts

```diff
@@ -1,26 +1,26 @@
 import ts from 'typescript';
-import { mockPluginParams, mockDiagnostic } from '../test-utils';
+import { mockPluginParams, mockDiagnostic, realPluginParams } from '../test-utils';
 import explicitAnyPlugin from '../../src/plugins/explicit-any';
 
 describe('explicit-any plugin', () => {
   it('adds explicit any', async () => {
-    const text = `import specifier1 from '../source';
```

Comment:
I can't find a way to make an import give an implicit any error. If the file doesn't exist or declarations don't exist, it's TS2307.

[TS playground link](https://www.typescriptlang.org/play?module=5&ts=4.1.2#code/JYWwDg9gTgLgBAZzAUwMbAGbGVAjHDKCEOAcgDpyB6BCAVylWVIG4g)